tests/bootstrap.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * WooCommerce Calypso Bridge Unit Tests Bootstrap
+ */
+class WC_Calypso_Bridge_Unit_Tests_Bootstrap {
+
+	/** @var WC_Calypso_Bridge_Unit_Tests_Bootstrap instance */
+	protected static $instance = null;
+
+	/** @var string directory where wordpress-tests-lib is installed */
+	public $wp_tests_dir;
+
+	/** @var string directory where WooCommerce tests are located */
+	public $wc_tests_dir;
+
+	/** @var string WooCommerce plugin directory */
+	public $wc_dir;
+
+	/** @var string testing directory */
+	public $tests_dir;
+
+	/** @var string plugin directory */
+	public $plugin_dir;
+
+	/**
+	 * Setup the unit testing environment.
+	 */
+	public function __construct() {
+		ini_set( 'display_errors','on' );
+		error_reporting( E_ALL );
+
+		// Ensure server variable is set for WP email functions.
+		if ( ! isset( $_SERVER['SERVER_NAME'] ) ) {
+			$_SERVER['SERVER_NAME'] = 'localhost';
+		}
+
+		$this->tests_dir    = dirname( __FILE__ );
+		$this->plugin_dir   = dirname( $this->tests_dir );
+		$this->wc_dir = dirname( $this->plugin_dir ) . '/woocommerce';
+		$this->wc_api_dev_dir = dirname( $this->plugin_dir ) . '/wc-api-dev';
+		$this->wc_tests_dir = dirname( $this->plugin_dir ) . '/woocommerce/tests';
+		$this->wp_tests_dir = getenv( 'WP_TESTS_DIR' ) ? getenv( 'WP_TESTS_DIR' ) : '/tmp/wordpress-tests-lib';
+
+		// load test function so tests_add_filter() is available
+		require_once( $this->wp_tests_dir . '/includes/functions.php' );
+
+		// load WC
+		tests_add_filter( 'muplugins_loaded', array( $this, 'load_wc' ) );
+
+		// load WC API Dev
+		tests_add_filter( 'muplugins_loaded', array( $this, 'load_wc_api_dev' ) );
+
+		// load WC Calypso Bridge
+		tests_add_filter( 'muplugins_loaded', array( $this, 'load_wc_calypso_bridge' ) );
+
+		// install WC
+		tests_add_filter( 'setup_theme', array( $this, 'install_wc' ) );
+
+		// load the WP testing environment
+		require_once( $this->wp_tests_dir . '/includes/bootstrap.php' );
+
+		// load WC testing framework
+		$this->includes();
+	}
+
+	/**
+	 * Load WooCommerce.
+	 */
+	public function load_wc() {
+		require_once( $this->wc_dir . '/woocommerce.php' );
+	}
+
+	/**
+	 * Load WC API Dev.
+	 */
+	public function load_wc_api_dev() {
+		define( 'WC_API_DEV_ENABLE_HOTFIXES', false );
```

Comment:
I think we don't need this anymore (here or in wc-api-dev). From a quick search, I don't think it's used anywhere.

Reply:
Indeed you are correct.